rpc-server/src/metrics.rs

```diff
@@ -96,12 +96,18 @@ lazy_static! {
         &["method_name"] // This declares a label named `method name`
     ).unwrap();
 
-    pub(crate) static ref REQUESTS_COUNTER: IntCounterVec = register_int_counter_vec(
-        "requests_counter",
+    pub(crate) static ref TOTAL_REQUESTS_COUNTER: IntCounterVec = register_int_counter_vec(
+        "total_requests_counter",
         "Total number of requests",
         &["request_type"] // This declares a label named `request_type`
     ).unwrap();
 
+    pub(crate) static ref TIMELINE_METHOD_REQUESTS_COUNTER: IntCounterVec = register_int_counter_vec(
```

Comment:
I think it'd be better to merge these two metrics `TOTAL_REQUESTS_COUNTER` and `TIMELINE_METHOD_REQUESTS_COUNTER` into one.

My assumption is that using Grafana we will be able to sum up the broken down metrics into a single value (if needed).

What do you think? Is there any benefit of having two similar yet different in details metrics?